.env.development

```diff
@@ -17,8 +17,8 @@ LANGUAGE_PREFERENCE_COOKIE_NAME=openedx-language-preference
 SITE_NAME=localhost
 MARKETING_SITE_BASE_URL=http://localhost:18000
 SUPPORT_URL=http://localhost:18000/support
-ORDER_HISTORY_URL=http://localhost:8140/orders/unified/order_history/
-RECEIPT_URL=http://localhost:8140/orders/unified/order_receipt/
+ORDER_HISTORY_URL=http://localhost:18130/api/v2/orders/
+RECEIPT_URL=http://localhost:18130/api/v2/checkout/receipt/
```

Comment:
If you decide to go with [this](https://github.com/openedx/frontend-app-ecommerce/pull/377#discussion_r1527713125) suggestion, then we can set an empty string as values for these config flags.